hack/build/update-base-image-shasums.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit -o nounset -o pipefail
+
+CONTAINERD_VERSION="1.4.0-106-gce4439a8"
+
+ARCHITECTURES=(
+    "amd64"
+    "arm64"
+    "ppc64le"
+)
+
+CONTAINERD_DEFAULT_VERSION="1.4.0-106-gce4439a8"
+read -r -p "What version of containerd? (defaults to $CONTAINERD_DEFAULT_VERSION) " CONTAINERD_VERSION
+CONTAINERD_VERSION=${CONTAINERD_VERSION:-${CONTAINERD_DEFAULT_VERSION}}
+
+CNI_DEFAULT_VERSION="v0.9.0"
+read -r -p "What version of CNI? (defaults to $CNI_DEFAULT_VERSION) " CNI_VERSION
+CNI_VERSION=${CNI_VERSION:-${CNI_DEFAULT_VERSION}}
+
+CRICTL_DEFAULT_VERSION="v1.19.0"
+read -r -p "What version of crictl: (defaults to $CRICTL_DEFAULT_VERSION) " CRICTL_VERSION
+CRICTL_VERSION=${CRICTL_VERSION:-${CRICTL_DEFAULT_VERSION}}
+
+echo
+CONTAINERD_BASE_URL="https://github.com/kind-ci/containerd-nightlies/releases/download/containerd-${CONTAINERD_VERSION}"
+for ARCH in "${ARCHITECTURES[@]}"; do
+    CONTAINERD_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}.linux-${ARCH}.tar.gz.sha256sum"
+    SHASUM=$(curl -sSL --retry 5 "${CONTAINERD_URL}" | awk '{print $1}')
+    ARCH_UPPER=$(echo "$ARCH" | tr '[:lower:]' '[:upper:]')
+    echo "ARG CONTAINERD_${ARCH_UPPER}_SHA256SUM=${SHASUM}"
+done
+
+echo
+# TODO (micahhausler): Once kind-ci/containerd-nightlies adds .sha256sum files, just fetch those
```

Comment:
I´m working on this, I hope to get it working today/tomorrow

Reply:
and here we go, we have shadsums for runc now https://github.com/kind-ci/containerd-nightlies/releases/tag/containerd-1.5.0-beta.0-36-g19ee068f9

Reply:
I'm hesitant to change the version of containerd with this PR, would you like me to update the script once a newer containerd is used in kind?

Reply:
ahh, yeah, sorry I didn´t mean to change it, just  wanted to check that the new sha5sum are right and works so we don´t have to change anything later